### ISSUE TITLE
"withRouteDefaults" failing with "options.response.schema"

### ISSUE THREAD

**Issue by YoannMa** [devinivy/toys#9]
I get a `Maximum call stack size exceeded` with that route, I believe `options.response.schema` is missing in https://github.com/hapipal/toys/blob/master/lib/index.js#L455

```javascript
const default = Toys.withRouteDefaults({
  options : {
    response : {
      schema : Joi.object()
    }
  }
});

default({
  method  : 'GET',
  path    : '/test',
  options : {
    response : {
      schema : Joi.array()
    }
  },
  handler : () => ['ok']
});
```

**Comment by devinivy** [comment]
D'oh!  That sounds right.  I'd definitely accept a patch.

**Comment by devinivy** [comment]
Resolved by #10 and published in v2.2.1.  Thanks!
